_posts/2023-05-24-windows-setup.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "5.24_NOTES_AnjaliN"
+tags: ["list", "of", "tags"]
+author: "Anjali Nuggehalli"
+---
+
+# Installation/Setting up Machine
+## AppInstaller and WinGet
```

Comment:
All Remaining headers should be promoted one level (level two to level one, level three to level two, etc.)